tests/test_edgeql_json.py

```diff
@@ -447,6 +439,53 @@ async def test_edgeql_json_accessor_22(self):
                 SELECT JT3.data[2]['b']['bar'][2]['bingo'];
             """)
 
+    async def test_edgeql_json_accessor_23(self):
+        await self.assert_query_result(
+            r'''select to_json('"hello"')[0] = <json>'h';''',
+            [True],
+        )
+
+        await self.assert_query_result(
+            r'''select to_json('"hello"')[-2] = <json>'l';''',
+            [True],
+        )
+
+    async def test_edgeql_json_accessor_24(self):
```

Comment:
I'd just combine test_edgeql_json_accessor_{24..27} into one test.

Reply:
We do this quite often. Historically I think it just helped getting a sense of which features worked and which still needed fixing/refining.

I can merge it in this case, since the error handling is introduced in one PR here.

Reply:
Let's merge into one -- the test suite will run a bit faster.